<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CoCOM</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="horizontalSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTextBrowser" name="outputTextBrowser">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QTabWidget" name="inputTabWidget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="basicSendTab">
        <attribute name="title">
         <string>basic</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="sendComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="basicSendButton">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="multiLineSendTab">
        <attribute name="title">
         <string>Muti Line</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="sendTextEdit"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="multiSendButton">
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="commandTab">
        <attribute name="title">
         <string>Command</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_23">
           <item>
            <widget class="QLineEdit" name="lineEdit_10"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_10">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLineEdit" name="lineEdit_7"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <widget class="QLineEdit" name="lineEdit_9"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_9">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLineEdit" name="lineEdit_8"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLineEdit" name="lineEdit_5"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QLineEdit" name="lineEdit_6"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="configToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="buttosToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openAction"/>
   <addaction name="clearAction"/>
   <addaction name="separator"/>
   <addaction name="actionPin"/>
   <addaction name="fileAction"/>
   <addaction name="toolsAction"/>
   <addaction name="settingAction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/pushpin.svg</normaloff>:/assets/icons/pushpin.svg</iconset>
   </property>
   <property name="text">
    <string>pin</string>
   </property>
  </action>
  <action name="openAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/play.svg</normaloff>:/assets/icons/play.svg</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="pauseAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/pause.svg</normaloff>:/assets/icons/pause.svg</iconset>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
  </action>
  <action name="stopAction">
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/stop.svg</normaloff>:/assets/icons/stop.svg</iconset>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/sync.svg</normaloff>:/assets/icons/sync.svg</iconset>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </action>
  <action name="settingAction">
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/sliders.svg</normaloff>:/assets/icons/sliders.svg</iconset>
   </property>
   <property name="text">
    <string>setting</string>
   </property>
  </action>
  <action name="toolsAction">
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/wrench.svg</normaloff>:/assets/icons/wrench.svg</iconset>
   </property>
   <property name="text">
    <string>tools</string>
   </property>
  </action>
  <action name="fileAction">
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/file.svg</normaloff>:/assets/icons/file.svg</iconset>
   </property>
   <property name="text">
    <string>file</string>
   </property>
  </action>
  <action name="sidebarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/assets/icons/side-bar.svg</normaloff>:/assets/icons/side-bar.svg</iconset>
   </property>
   <property name="text">
    <string>sidebar</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
